codersdk/templates.go

```diff
@@ -34,6 +34,7 @@ type UpdateActiveTemplateVersion struct {
 }
 
 type UpdateTemplateMeta struct {
+	Name                       string `json:"name,omitempty"`
```

Comment:
This should have `validate:"username"` added to it! Otherwise, we could get invalid strings as usernames.

The test you broke will fail too, because that isn't actually a valid template name.

Reply:
Fixed